src/javascript/SelectorTypes/Picker/PickerDialog/RightPanel/ContentLayout/ContentTable/ContentTable.jsx

```diff
@@ -113,9 +114,10 @@ export const ContentTable = ({
 
     useEffect(() => {
         if (selection.length > 0) {
-            const toRemove = selection.filter(s => paths.indexOf(s.path) === -1).map(s => s.path);
+            const toRemove = selection.filter(s => paths.indexOf(s.path) === -1);
             if (toRemove.length > 0) {
-                dispatch(reduxActions.removeSelectionAction(toRemove));
+                // TODO why do we have this?
+                // dispatch(reduxActions.removeSelectionAction(toRemove));
```

Comment:
That was something in jcontent, to remove selection (preview) if an item is not present in the list anymore. But that does not make sense in picker (where selection is persistent)

Reply:
Ok, I will remove it then.